lib/dynflow/config.rb

```diff
@@ -115,6 +115,14 @@ def method_missing(name)
       { 'hostname' => Socket.gethostname, 'pid' => Process.pid }
     end
 
+    config_attr :backup_deleted_plans, Algebrick::Types::Boolean do
```

Comment:
I believe not. `backup_dir` nil mean use the default dir. Option for backup on/off seems reasonable. I'll recheck when the foreman-tasks part is added